vendor/engines/secure_rooms/app/presenters/secure_rooms/account_presenter.rb

```diff
@@ -0,0 +1,23 @@
+module SecureRooms
+
+  class AccountPresenter < SimpleDelegator
+
+    def self.wrap(accounts)
+      accounts.map { |account| new(account) }
+    end
+
+    def to_json
+      {
+        id: id,
+        type: type,
+        description: description,
+        account_number: account_number,
+        expiration_month: expiration_month,
+        expiration_year: expiration_year,
+        name: owner_user.try(:full_name)
```

Comment:
There's a validation on Account so there should always be an owner. However, I have seen it happen a couple times where there is no owner (due to a manual deletion of a user--which can only be done from the console/directly in the database, or the result of a bug).